_artifacts/logstash-oss-with-opensearch-output-plugin/logstash-oss-with-opensearch-output-plugin-7.16.3-docker-arm64.markdown

```diff
@@ -7,5 +7,15 @@ architecture: arm64
 slug: logstash-oss-with-opensearch-output-plugin-7.16.3-docker-arm64
 category: logstash-oss-with-opensearch-output-plugin
 type: docker_hub
```

Comment:
It might be good to rename this constant to just `docker` since this is supporting both DockerHub and ECR.